backend/app/views/spree/admin/shared/_no_objects_found.html.erb

```diff
@@ -1,4 +1,4 @@
 <%= Spree.t(:no_resource, resource: plural_resource_name(resource)) %>
-<% if can? :create, resource %>
+<% if can? :create, resource and !new_resource_url.nil? %>
```

Comment:
We should use `local_assigns` here to make it optional.

```rb
if can?(:create, resource) && local_assigns[:new_resource_url]
```

Reply:
Yeah, is more clean like you propose :+1: 

I'll update the pulll request accordingly